frontend/src/Components/Profile.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from 'react'
+import { Login } from 'Components/Login'
+
+const URL = 'https://auth-login-project.herokuapp.com/users'
+
+export const Profile = (loggedInUser) => {
+  // loggedInUser = loggedInUser.loggedInUser
+  const [userId, setUserId] = useState(0)
+  const [accessToken, setAccessToken] = useState('')
+  console.log(loggedInUser)
+
+  const [logout, setLogout] = useState(false)
+
+
+  useEffect(() => {
+    fetch(`${URL}/${userId}`, {
+      method: 'GET',
+      headers: { 'Authorization': accessToken }
+    })
+      .then(res => res.json())
+      .then(json => console.log(json))
```

Comment:
maybe not a good idea to leave this console log here?
try to store that user name from the json and print out out in the h1 or something instead?